container/container.go

```diff
@@ -405,21 +405,20 @@ func Subnet(cnr Container) (res subnetid.ID) {
 	return
 }
 
-const attributeHomoHashEnabled = "true"
-
 // DisableHomomorphicHashing sets flag to disable homomorphic hashing of the
 // Container data.
 //
 // See also IsHomomorphicHashingDisabled.
-func DisableHomomorphicHashing(cnr *Container) {
-	cnr.SetAttribute(container.SysAttributeHomomorphicHashing, attributeHomoHashEnabled)
+func (x *Container) DisableHomomorphicHashing() {
```

Comment:
Lets leave it a function.

Reply:
why? it now uses private fields

Reply:
It previously did not, lets retain how it was.

Reply:
why do you want to do that via `SetAttribute` and not via `api-go` methods?

Reply:
Now because it is already done, later `api-go` won't have these methods.

Reply:
> won't have these methods

why?

Reply:
* https://github.com/nspcc-dev/neofs-api-go/issues/395